src/optionals/datetime.c

```diff
@@ -129,30 +185,291 @@ static Value strftimeNative(DictuVM *vm, int argCount, Value *args) {
 }
 
 #ifdef HAS_STRPTIME
```

Comment:
Wanna watch this pre-processor

Reply:
It would also be nice to have strptime, strings to dates will def be needed

Reply:
Was going to add that at in this PR but wanted to make sure the overarching approach was acceptable first. :D